addons/mass_mailing/static/src/scss/mass_mailing.wysiwyg.scss

```diff
@@ -18,24 +18,37 @@
     }
 }
 
+@mixin email_designer_btn {
+    align-items: center;
+    width: 24px;
+    height: 24px;
+    background-color: #337ab7 !important;
+    border: 1px solid #2e6da4 !important;
```

Comment:
Can't you do without this important ? Like can't you modify a selector so that it gains priority and we can avoid that ?

Reply:
@tde-banana-odoo Do we need to change background color on hover ? 

In the template of `o_we_website_top_actions` there are `btn-secondary` classes has been added in the classlist of these buttons.If we remove `!important` here, then hovering on the button will override the background color.
It will work without `!important` but when button is disabled or clicked then background color will be from `btn-secondary`.

Reply:
Well I guess that's ok, thanks for explanation :)